Fix MicroProfile JWT related issues on JDK 21 based executions

There are quite some tests stable failures when the TS is executed on JDK 21.
Mostly it seem due to an encryption algorithm used by the `jose4j` library, which seems to be not supported by the runtime, e.g.:

```
java.security.spec.InvalidKeySpecException: java.security.InvalidKeyException: Unable to decode key
	at jdk.crypto.ec/sun.security.ec.ECKeyFactory.engineGeneratePrivate(ECKeyFactory.java:168)
	at java.base/java.security.KeyFactory.generatePrivate(KeyFactory.java:389)
	at org.jboss.eap.qe.microprofile.jwt.auth.tool.AbstractKeyLoader.readPrivateKey(AbstractKeyLoader.java:55)
...
Caused by: java.security.InvalidKeyException: Unable to decode key
	at java.base/sun.security.pkcs.PKCS8Key.<init>(PKCS8Key.java:98)
	at jdk.crypto.ec/sun.security.ec.ECPrivateKeyImpl.<init>(ECPrivateKeyImpl.java:81)
	at jdk.crypto.ec/sun.security.ec.ECKeyFactory.implGeneratePrivate(ECKeyFactory.java:243)
	at jdk.crypto.ec/sun.security.ec.ECKeyFactory.engineGeneratePrivate(ECKeyFactory.java:164)
	... 123 more
Caused by: java.io.IOException: extra data at the end
	at java.base/sun.security.util.DerValue.<init>(DerValue.java:432)
	at java.base/sun.security.util.DerValue.<init>(DerValue.java:344)
	at java.base/sun.security.pkcs.PKCS8Key.<init>(PKCS8Key.java:96)
	... 126 more	

```

This issue should track the adjustments to the test suite in order to run MicroProfile JTW tests green on JDK 21, too.

--- a/tooling-mp-jwt-auth-tool/src/main/java/org/jboss/eap/qe/microprofile/jwt/auth/tool/AbstractKeyLoader.java
+++ b/tooling-mp-jwt-auth-tool/src/main/java/org/jboss/eap/qe/microprofile/jwt/auth/tool/AbstractKeyLoader.java
@@ -46,7 +46,7 @@ public abstract class AbstractKeyLoader<PRIVATE_KEY, PUBLIC_KEY> {
             String privateKeyPEM = key.toString()
                     .replace("-----BEGIN PRIVATE KEY-----", "")
                     .replaceAll(System.lineSeparator(), "")
-                    .replace("-----BEGIN PRIVATE KEY-----", "");
+                    .replace("-----END PRIVATE KEY-----", "");
 
             byte[] encoded = Base64.decodeBase64(privateKeyPEM);
 

